README.md

```diff
@@ -35,25 +35,32 @@ const Koa = require('koa')
 const app = new Koa()
 
 const router = require('koa-router')()
-const WebMonetization = require('koa-web-monetization')
-const monetization = new WebMonetization()
-
+const { WebMonetizationMiddleWare, KoaWebMonetization } = require('koa-web-monetization')
+const monetizer = new KoaWebMonetization()
+const EXPRESS_WEB_MONETIZATION_CLIENT_PATH =  '/node_modules/express-web-monetization/client.js'
 // This is the SPSP endpoint that lets you receive ILP payments.  Money that
 // comes in is associated with the :id
-router.get('/pay/:id', monetization.receiver())
+router.get(monetizer.receiverEndpointUrl, monetizer.receive.bind(monetizer))
 
 // This endpoint charges 100 units to the user with :id
 // If awaitBalance is set to true, the call will stay open until the balance is sufficient. This is convenient
 // for making sure that the call doesn't immediately fail when called on startup.
-router.get('/content/:id/:content_id/', monetization.paid({ price: 100, awaitBalance: true }), async ctx => {
-  // load content by :content_id
+router.get('/content/', async ctx => {
+
+  await ctx.state.awaitBalance(100)
+  ctx.state.spend(100)
+  // load content
 })
 
 router.get('/', async ctx => {
   // load index page
 })
 
+router.get('/scripts/client.js', async ctx => {
+  ctx.body = await fs.readFile(path.join(__dirname, EXPRESS_WEB_MONETIZATION_CLIENT_PATH);
+}
 app
+  .use(compose([WebMonetizationMiddleWare(monetizer), router.middleware()]))
```

Comment:
I think `compose` needs to be required. How come you can't do `use(WebMonetizationMiddleware(monetizer))` followed by `use(router.routes())`?

Reply:
I think this is because when I don't composing WebMonetizationMiddleware with router.middleware, ctx.state does not get populated with the spend and awaitBalance functions.